Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8 -Duser.language=en-US -Duser.country=US -Duser.language=en
org.apache.commons.math3.complex.ComplexTest#testEqualsImaginaryDifference
.
Time: 0.954

OK (1 test)

org.apache.commons.math3.complex.ComplexTest#testEqualsRealDifference
.
Time: 0.078

OK (1 test)

org.apache.commons.math3.complex.ComplexTest#testEqualsClass
.
Time: 0.022

OK (1 test)

org.apache.commons.math3.analysis.solvers.LaguerreSolverTest#testQuadraticFunction
.
Time: 0.08

OK (1 test)

org.apache.commons.math3.analysis.solvers.LaguerreSolverTest#testQuinticFunction
.
Time: 0.348

OK (1 test)

org.apache.commons.math3.analysis.solvers.LaguerreSolverTest#testQuinticFunction2
.
Time: 1.089

OK (1 test)

org.apache.commons.math3.complex.ComplexTest#testEqualsNull
.
Time: 0.016

OK (1 test)

org.apache.commons.math3.complex.ComplexTest#testReciprocalZero
.E
Time: 0.065
There was 1 failure:
1) testReciprocalZero(org.apache.commons.math3.complex.ComplexTest)
java.lang.AssertionError: expected:<(NaN, NaN)> but was:<(Infinity, Infinity)>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at org.apache.commons.math3.complex.ComplexTest.testReciprocalZero(ComplexTest.java:334)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at fr.inria.lille.repair.MethodTestRunner.runTest(MethodTestRunner.java:44)
	at fr.inria.lille.repair.MethodTestRunner.access$0(MethodTestRunner.java:37)
	at fr.inria.lille.repair.MethodTestRunner$1.call(MethodTestRunner.java:20)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

FAILURES!!!
Tests run: 1,  Failures: 1

